doc/man3/OSSL_trace_set_channel.pod

```diff
@@ -178,9 +178,15 @@ Traces BIGNUM context operations.
 
 =back
 
-There is also C<OSSL_TRACE_CATEGORY_ANY>, which works as a fallback
+There is also C<OSSL_TRACE_CATEGORY_ALL>, which works as a fallback
 and can be used to get I<all> trace output.
 
+Note however that in this case all trace output will effectively be
```

Comment:
Comma after however?

Reply:
Hugh, we are entering a mined field! I googled a little bit and it seems that punctuation purists require two commas (see e.g. [here](https://www.thepunctuationguide.com/period.html) and [here](https://www.dailywritingtips.com/how-to-punctuate-introductory-phrases)).

> Note, however, that ...

Our code base is undecided:

```make
$ rgrep -i 'note[, ]*however'
crypto/aes/asm/aes-s390x.pl:83:# Note however that it's disengaged, because it was measured to
todo/opensslrng-audit-notes.txt:111:source to use. Note however, that it does *not* relax the requirement for seeding the DRBG properly, it only disables
doc/man3/EC_GFp_simple_method.pod:39:optimised implementations for the NIST P224, P256 and P521 curves respectively. Note, however, that these
doc/man3/RAND_DRBG_get0_master.pod:44:they are thread-local. Note however, that changes to these two instances
doc/man3/OSSL_trace_set_channel.pod:184:Note however that in this case all trace output will effectively be
```

I think I'll choose the two comma variant, it looks _very_ intellectual ;-)

Reply:
Fixed in 90b7ffc3d02cb2060456559a0a175731343987fe.

Reply:
To commas are fine.  When I read I only put one pause in but I'm aware that two is common.

The other option would have been "However, note"